frontend/screens/inbox/InboxInterface.js

```diff
@@ -25,90 +39,92 @@ const InboxInterface = () => {
         setUserId(id)
         setUserType('brand')
       }
     }
-    getData(); 
+    getData();
   }, []);
 
   const handleSearchChange = (text) => {
     setSearchQuery(text);
   };
   return (
-    <ScrollView contentContainerStyle={styles.scrollViewContent}>
-      <View style={[styles.inboxContainer, { height: Dimensions.get("window").height }]}>
-        <View style={styles.headerContainer}>
-          <View style={styles.headerContent}>
-            <TouchableOpacity onPress={() => navigation.navigate("UserProfile")}>
-              <View style={styles.headerTitleContainer}>
-                <Text style={styles.headerTitle}>Inbox</Text>
-              </View>
-            </TouchableOpacity>
-            <TouchableOpacity onPress={() => navigation.navigate("NewMessage")}>
-              <View style={styles.newMessageButton}>
-                <Image
-                  style={styles.newMessageIcon}
-                  contentFit="cover"
-                  source={require("../../assets/depth-5-frame-021.png")}
-                />
-              </View>
-            </TouchableOpacity>
-          </View>
+    <View style={{ width: "100%", height: "100%" }}>
+      <View style={styles.headerContainer}>
+        <View style={styles.headerContent}>
+          <TouchableOpacity onPress={() => navigation.navigate("UserProfile")}>
+            <View style={styles.headerTitleContainer}>
+              <Text style={styles.headerTitle}>Inbox</Text>
+            </View>
+          </TouchableOpacity>
+          <TouchableOpacity onPress={() => navigation.navigate("NewMessage")}>
+            <View style={styles.newMessageButton}>
+              <Image
+                style={styles.newMessageIcon}
+                contentFit="cover"
+                source={require("../../assets/depth-5-frame-021.png")}
+              />
+            </View>
+          </TouchableOpacity>
         </View>
-        <View style={styles.searchContainer}>
-          <View style={styles.searchContent}>
-            <Image
-              style={styles.searchIcon}
-              contentFit="cover"
-              source={require("../../assets/depth-3-frame-04.png")}
-            />
-            <TextInput
-              style={styles.searchInput}
-              placeholder="Search for a project, brand or Influencer"
-              placeholderTextColor={Color.colorSlategray_200}
-              value={searchQuery}
-              onChangeText={handleSearchChange}
-            />
+      </View>
+      <ScrollView onScroll={handleScroll} contentContainerStyle={styles.scrollViewContent}>
+        <View style={[styles.inboxContainer, { height: Dimensions.get("window").height }]}>
+          <View style={styles.searchContainer}>
+            <View style={styles.searchContent}>
+              <Image
+                style={styles.searchIcon}
+                contentFit="cover"
+                source={require("../../assets/depth-3-frame-04.png")}
```

Comment:
here also, the name of the file `depth-3-frame-04` isn't very informative, can you rename the file so that ut clearly states what's inside it?